Extracting reference data (clustering).ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<p>Comment should be "Selecting New Reference Data With K-Means Clusterin"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Thanks for noticing. Forgot to change after copy paste. 

Fixed and pushed to extracting-reference-data branch